### PR TITLE
Allow insecure registries

### PR DESCRIPTION
**Pull request by davidcassany**
This PR allows to pull images from insecure registries with the flag `--tls-verify`. 

Part of #2132

### CODE REVIEW

File: cmd/flags.go [hunk 1]
```diff
@@ -78,6 +79,11 @@ func addLocalImageFlag(cmd *cobra.Command) {
 	cmd.Flags().Bool("local", false, "Use an image from local cache")
 }
 
+// addVerifyRegistryFlag add local image flag shared between install, pull-image, upgrade
+func addTLSVerifyFlag(cmd *cobra.Command) {
+	cmd.Flags().Bool("tls-verify", false, "Require HTTPS and verify certificates of registries (default: true)")
```
Comment (anmazzotti) [inline cmd/flags.go:84]: Flag is false by default, not sure if the description is wrong or the actual default, I like it to be on by default or at least that's what I would expect as user.

**Review by anmazzotti — CHANGES_REQUESTED** [review]
Actual default should be true I believe. 
Also I don't fully get how this is supported now, I recall we didn't do verification before, is this a new feature of `github.com/google/go-containerregistry`?

File: cmd/flags.go (hunk 1 above)
Comment (davidcassany) [reply cmd/flags.go]: Done

**Review by anmazzotti — APPROVED** [review]
Awesome feature!
Looks good.

**Pull request merged**